### PR TITLE
Dry certification config

### PR DESCRIPTION
**Pull request by ahmaxed**
ref #39284
ref https://github.com/freeCodeCamp/freeCodeCamp/pull/39695

### CODE REVIEW

**Comment by ahmadabdolsaheb** [comment]
@ojeytonwilliams should we produce these cert maps when building the curriculum?

**Comment by ojeytonwilliams** [comment]
@ahmadabdolsaheb yes, I'd say so.  Shaun's been working on https://github.com/freeCodeCamp/freeCodeCamp/pull/39695 which is looking to do that.  For the 'building' part, we still need to work out how to keep the ability to quickly live-reload challenges in the browser whenever you change a challenge.  If we require the entire curriculum to be reloaded, this makes the process very slow and frustrating.  I'm not sure how best to solve this, yet.

Also, I think we should consider using `meta.yml` over `meta.json`, simply because we repeat the titles in the meta.json purely to help humans order the challenges correctly.  Effectively they're comments masquerading as code, which is another source of confusion.

File: config/misc.js [hunk 1]
```diff
@@ -1,2 +1,4 @@
 exports.oldDataVizId = '561add10cb82ac38a17513b3';
```
Comment (raisedadead) [inline config/misc.js:1]: This should be moved to the area other cert-ids are defined.

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Review by raisedadead — COMMENTED** [review]
Mind resolving the conflict on this PR?

**Comment by ahmadabdolsaheb** [comment]
Since the completion hours are read from the translations, it seems that they fine not being pulled from the config.

**Comment by ojeytonwilliams** [comment]
So, it's kind of annoying that I could introduce a bug (fixed by the last commit) that didn't cause any tests to fail.  It's in `randomAPIs` though, so maybe it's not a huge deal.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1: 

I did rebase and add a commit, though, so another review would be good.

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
@nhcarrigan please review and merge when you get a chance.

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me.

Testing locally: claiming cert, attempting to claim cert w/o projects, and viewing cert all appear to function as intended.

**Pull request merged**